kuma/wiki/tests/test_views.py

```diff
@@ -1361,9 +1358,15 @@ def _createAndRunTests(slug):
             locale = settings.WIKI_DEFAULT_LANGUAGE
             foreign_locale = 'es'
             new_doc_url = reverse('wiki.new_document')
-            invalid_slug = invalid_slug1 = "some/thing"
-            invalid_slug2 = "some?thing"
-            invalid_slug3 = "some thing"
+            invalid_slug = "some/thing"
```

Comment:
Do we still need this `invalid_slug` varaible? Or can we use `invalid_slugs[0]` below?